test/integration/features/cmd-addons.feature

```diff
@@ -8,12 +8,9 @@ Feature: Addons command and its subcommands
       And executing "minishift add-ons" fails
       And executing "minishift add-on" fails
 
- Scenario: Installing default add-ons manually
-     When executing "minishift addons install --defaults" succeeds
-     Then stdout should contain
-      """
-      Default add-ons anyuid, admin-user, xpaas, registry-route installed
-      """
+  @minishift-only
+  Scenario: Default add-ons are installed but not enabled by default
+  Note: default addons were installed during previous commands automatically.
```

Comment:
Maybe you could indent this so that it becomes clear that it is part of the scenario description